recipes/pymannkendall/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pymannkendall" %}
+{% set version = "1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a10a1cc24c7466bce7e1c0c70c5601ad44aee573d2ce161e6e68b60f6a94e11e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - numpy
```

Comment:
Usually only `python` and `pip` should be in the `host` section. Unless they are really needed for building, I would suggest removing `numpy` and `scipy` from here.

Reply:
My understanding was that conda-forge wanted run-time requirements in the host section (as opposed to bioconda, where they're removed).

Reply:
I am not aware of that, and based on few reviews I have seen and https://conda-forge.readthedocs.io/en/latest/meta.html#use-pip (the last paragraph), I thought the opposite was the case. However, I am ok with merging the recipe either way.

Reply:
Let's just merge it then, it won't have any real tangible effect which way we handle the `host` section :)